src/Core/Content/SubscriptionToProduct/SubscriptionToProductDefinition.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+namespace Kiener\MolliePayments\Core\Content\SubscriptionToProduct;
+
+use Shopware\Core\Content\Product\ProductDefinition;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityDefinition;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\CreatedAtField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\CustomFields;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\DateTimeField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\FkField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\ApiAware;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\PrimaryKey;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\Flag\Required;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\FloatField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\IdField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\StringField;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\UpdatedAtField;
+use Shopware\Core\Framework\DataAbstractionLayer\FieldCollection;
+use Shopware\Core\System\SalesChannel\SalesChannelDefinition;
+use Shopware\Core\Framework\DataAbstractionLayer\Field\JsonField;
+
+class SubscriptionToProductDefinition extends EntityDefinition
+{
+    /**
+     * @return string
+     */
+    public function getEntityName(): string
+    {
+        return 'mollie_subscription_to_product';
+    }
+
+    /**
+     * @return string
+     */
+    public function getEntityClass(): string
```

Comment:
- Not added any association for productId and salesChannel Id. So if it is needed, then please add it.
https://developer.shopware.com/docs/guides/plugins/plugins/framework/data-handling/add-data-associations